sockeye/model.py

```diff
@@ -109,10 +109,13 @@ def __init__(self, config: ModelConfig, prefix: str = '') -> None:
 
         # source & target embeddings
         embed_weight_source, embed_weight_target, out_weight_target = self._get_embed_weights(self.prefix)
-        self.embedding_source = encoder.Embedding(self.config.config_embed_source,
-                                                  prefix=self.prefix + C.SOURCE_EMBEDDING_PREFIX,
-                                                  embed_weight=embed_weight_source,
-                                                  is_source=True)
+        if isinstance(self.config.config_embed_source, encoder.PassThroughEmbeddingConfig):
+            self.embedding_source = encoder.PassThroughEmbedding(self.config.config_embed_source) # type: encoder.Encoder
```

Comment:
[super nit]: pep8: two spaces between code and `#` from comments.

Reply:
I know 😄 . I missed this one